godtools/ViewControllers/Platform/GTLanguageSettingsViewController.xib

```diff
@@ -0,0 +1,140 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
What I like to do for constraints are pick the top most item and give it a center-Y constraint with a 0 constant and a multiplier that puts the icon where it should go for an iPhone 6. This constraint should have a medium priority.

I then do a vertical spacing constraint from that to the item below it, and another between the 2nd and third items, and so on and so forth.  

Finally I add a bottom space constraint from the last item to the container bottom and a top space from the first item to the container top with highest priority to make sure nothing gets pushed off the top of bottom of the view.

These constraints make sure the layout looks great on iPhone 6 and 6 Plus, but also lets it shift around for smaller devices. It may not be perfectly aligned, but everything should be on screen and not overlapping.